### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by ghsdh3409**

### CODE REVIEW

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java [hunk 1]
```diff
@@ -84,6 +86,27 @@ public void prepare(final String fileName, final Integer key, final ReadableMap
     
     player.setAudioStreamType(audioStreamTypeFinal);
 
+    if (module.category != null) {
```
Comment (ghsdh3409) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java]: @ohtangza Oh! Setting `category` is supported here!

Comment (ohtangza) [reply android/src/main/java/com/zmxv/RNSound/RNSoundModule.java]: Oh, that's true.

File: index.d.ts [hunk 2]
```diff
@@ -206,3 +211,5 @@ export default class Sound {
    */
   isPlaying(): Promise<boolean>
 }
+
+export = Sound;
```
Comment (ohtangza) [inline index.d.ts:215]: I think we also need to add the information about `setSpeed` (Android version condition), `isPlaying`, `isHeadsetPlugged`, `setStreamMute`, `registerHeadsetPlugChangeListener`, `unregisterHeadsetPlugChangeListener`, `setCategory` (added bluetooth option)

Comment (ohtangza) [reply index.d.ts:215]: Made issue: #28.

**Review by ohtangza — APPROVED** [review]
Looks all good.

**Comment by ohtangza** [comment]
Let's make a pull request onto upstream after merging this, yeah!

**Pull request merged**